calendar.html

```diff
@@ -107,6 +107,10 @@ <h3 style="text-align:center;color:aliceblue;font-size: 1.6rem;">
     animation: load6 1.7s infinite ease, round 1.7s infinite ease;
   }
 
+.fc-unthemed td.fc-today{
+  background: var(--brand-color-primary) !important;
```

Comment:
Use --brand-color-secondary instead of primary, it works better with the white text when switched to dark mode